app/src/main/kotlin/com/wire/android/ui/home/HomeScreen.kt

```diff
@@ -41,9 +41,9 @@ fun HomeScreen(startScreen: String?, viewModel: HomeViewModel) {
             )
         }
 
-        NavigationDrawer(
-            drawerContainerColor = MaterialTheme.colorScheme.surface,
-            drawerTonalElevation = 0.dp,
+        ModalDrawer(
```

Comment:
Have you tried this new `ModalNavigationDrawer` from material3? 😉

Reply:
Ok, they still haven't addressed it yet, we don't know when they will fix it, so it might actually be worth doing a downgrade to older material